_posts/2023-02-22-#25038.md

```diff
@@ -7,18 +7,275 @@ authors: [glozow]
--->
 
+1. Did you review the PR? [Concept ACK, approach ACK, tested ACK, or NACK](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)?
+What was your review approach?
+
+### Concept
+
+1. What is the difference between a transaction that "explicitly" signals BIP125 and one that "inherits" BIP125
+   signaling? Why is it impossible for a v3 transaction to be one but not the other?
+
+1. Why should there be a rule to check whether a replacement transation is more incentive compatible to mine?
+
+1. Is min(package feerate, individual feerate) an accurate incentive compatibility score for v3 transactions? What
+   about in general? (See `CheckMinerScores()` [implementation](https://github.com/bitcoin-core-review-club/bitcoin/commit/58e811ecb1e1977422ecda2af62460e8efc057be)).
+
+1. Imagine that there are two conflicting, txA and txB, both 2000vB. We *really* need one of them to confirm
+   immediately, but it doesn't matter which one. A malicious counterparty has already broadcast transaction txB which
+pays 10,000sat in fees (5sat/vB). We also know that it's guaranteed (just pretend, ok?) that anything above 20sat/vB
+will be mined immediately, while anything below 20sat/vB will not.
+	- Let's say txA and txB are non-v3 transactions. What fee can we put on txA to *guarantee* that either txA or
+	  txB will reach a package feerate of 20sat/vB? (Hint: txA needs to replace txB and any descendants it may have.
+	  In the non-v3 world, txA can have up to 99KvB of descendants. What is the maximum fee the counterparty can
+	  attach to descendants of txB without bumping it to 20sat/vB?)
+	- Now imagine txA and txB are both v3 transactions. What fee can we put on txA to guarantee that either txA or txB will
+	  reach a package feerate of 20sat/vB? (Hint: in the v3 world, txA cannot have more than 1000vB of descendants).
+
+1. If all mining nodes on the network are running with this v3 policy, while none of the non-mining nodes are, what happens?
+
+1. If 10% of nodes on the network are running with this v3 policy, while we haven't upgraded our node yet, what happens?
+   What about 99% of nodes?
+
+1. If we were running a business that accepted unconfirmed transactions as payments, what changes should we make so that our wallet correctly tracks v3 transactions?
+
+1. Who might be interested in using v3 transactions? Who probably wouldn't be interested?
+
+1. If there are some wallets/users/applications that use v3 while others don't, is there a privacy issue (i.e. people
+   can fingerprint which wallet likely created a transaction by looking at its version)?
+
+1. After the addition of v3, all non-v3 transactions are still subject to the existing limitations and pinning attacks.
+   Wouldn't it be safer to enforce these rules for all transactions on the network?
+
+1. V3 limits each transaction to 1 parent or 1 child (2 generations only), and perhaps that seems
```

Comment:
I just realized - we never explicitly specify "parent" or "child" to refer to _unconfirmed_ transactions only, and I think that may be confusing to some people. Could qualify it everywhere, or slap a big disclaimer at the beginning of the notes (and probably repeat for the Questions section)?